packages/components/src/internal/components/editable/utils.test.ts

```diff
@@ -10,7 +10,7 @@ import { makeTestQueryModel } from '../../../public/QueryModel/testUtils';
 
 import { QueryInfo } from '../../../public/QueryInfo';
 
-import { QueryColumn } from '../../../public/QueryColumn';
+import { QueryColumn, QueryLookup } from '../../../public/QueryColumn';
```

Comment:
More of a question now that you've completed this initial pass. How do you feel the test coverage is in this file? Are there areas left uncovered? Are there new things introduced that we can circle back on to add coverage for here?

Reply:
For utils I don't know that we need additional coverage. In theory we could add more test coverage for event handling like we have for addRows, addColumns, changeColumn, etc, however we do have test coverage for those events via our selenium tests so I am not sure how important it is to add coverage here.

We could also add more coverage for actions, like what we have for fillCell, however, like the utils stuff, we have a ton of selenium coverage, so I am not sure how much of a priority it really is.

Same would apply for the methods I added to the EditorModel, we could have some test coverage for them all, but again, we have a ton of selenium coverage for all of those methods.